packages/mdc-drawer/README.md

```diff
@@ -70,6 +70,131 @@ CSS classes:
 | `mdc-permanent-drawer__content`        | Mandatory. Needs to be set on the container node for the drawer content.   |
 | `mdc-permanent-drawer__toolbar-spacer` | Optional. Add to node to provide the matching amount of space for toolbar. |
 
+## Persistent drawer usage
+
+A persistent drawers can toggle open or closed. The drawer sits on the same surface elevation as the content. It is closed by default. When the drawer is outside of the page grid and opens, the drawer forces other content to change size and adapt to the smaller viewport. Persistent drawers stay open until closed by the user.
```

Comment:
Maybe change this first sentence to "Persistent drawers can be toggled open or closed"?